### ISSUE TITLE
Make task backgroundable

### ISSUE THREAD

**Issue by lyuben-todorov** [ciselab/TestGenie#44]
# Description
Make the evosuite runner class leverage Task.Backgroundable so that there's a progress indicator shown.
# Definition of Done

**Comment by martinmladenov** [comment]
Backgroundable tasks often allow the user to cancel the task. Should we also allow that?

**Comment by lyuben-todorov** [comment]
> Backgroundable tasks often allow the user to cancel the task. Should we also allow that?

Yep, good idea, this has been implemented. Thankfully the evosuite process is cooperative and doesn't ignore the kill signals.
